components/admin_console/permission_schemes_settings/edit_post_time_limit_modal/index.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
maybe we can remove the index file completely since we only need the config and one dispatch.
This can be done using hooks and is way simpler than connecting the whole thing to the store.

see https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/license_settings/trial_banner/trial_banner.tsx on how to use that.

Reply:
ok, in that case we can leave it as is, although i am 4/5 that making that component use hooks instead makes it way more readable and testing should not be that much of a problem ... for the sake of not blocking this any longer I am fine with the solution right now